src/stories/containers/cu-about/cu-about.api.ts

```diff
@@ -0,0 +1,219 @@
+import { request, gql } from 'graphql-request';
+import { GraphQlEndpoint } from '../../../config/endpoint';
+import { CuCategoryEnum } from '../../../core/enums/cu-category.enum';
+import { CuStatusEnum } from '../../../core/enums/cu-status.enum';
+import { LinkTypeEnum } from '../../../core/enums/link-type.enum';
+import { LinkModel } from '../../components/cu-table-column-links/cu-table-column-links';
+import { CoreUnit } from '../../components/title-navigation-cu-about/title-navigation-cu-about';
+
+interface BudgetStatementFTEs {
+  month: string;
+  ftes: number;
+}
+
+interface BudgetStatement {
+  budgetStatementFTEs: BudgetStatementFTEs[];
+}
+
+export enum CuMipStatus {
+  RFC = 'RFC',
+  FORMAL = 'Formal',
+  SUBMISSION = 'Submission',
+  Accepted = 'Accepted',
+  Rejected = 'Rejected',
+  Obsolete = 'Obsolete',
+}
```

Comment:
Let's keep the enums camel cased like all the others defined in the src/core/enums folder